File: Java/Math/FibonacciGenerator.java
import java.util.ArrayList;
import java.util.Arrays;

/*
 * @author Jared Scott
 * 
 */
 
public class FibonacciGenerator {
	/*
	 * This class will print a list containing a specified amount of fibonacci numbers  
	 * ARGUMENTS:
	 * 	Integer: The amount of fibonacci numbers you would like to generate 
	 *
	 * OUTPUT:
	 *	Printed Array: An array holding the desired number of fibonacci numbers 
	 */
	public static void main(String[] args) {
		ArrayList<Integer> fibNums = new ArrayList<Integer>();
		fibNums.add(1);
		fibNums.add(2);
		int curNum = 2;
		while (curNum < Integer.parseInt(args[0])) {
			fibNums.add(fibNums.get(fibNums.size()-2) +fibNums.get(fibNums.size()-1));
			curNum += 1;
		}
		System.out.println(fibNums);
	}
}

File: Python/Math/fibonacciGenerator.py
#@author Jared Scott

def fibonacci(values):
    '''
        This function will return a list containing 'values' number of fibonacci numbers. 
        
        Arguments: 
            (Integer) values: the number of fibonacci numbers you would like to generate 
            
        Output:
            A list containing the specified number of fibonacci numbers 
    '''
    fibNums = []
    fibNums.append(1) 
    fibNums.append(2) 
    while len(fibNums) < values:
        fibNums.append(fibNums[-2] + fibNums[-1])
    return fibNums
    
print(fibonacci(20))